hack/generate/csv.sh

```diff
@@ -135,6 +136,9 @@ kafka_image "knative-kafka-storage-version-migrator__migrate"    "${KNATIVE_EVEN
 
 image 'KUBE_RBAC_PROXY'          "${rbac_proxy}"
 image 'KN_PLUGIN_EVENT_SENDER'   "${kn_event}-sender"
+image 'KN_CLIENT'                "${registry}/knative-v$(metadata.get dependencies.cli):knative-client"
+image 'KN_PLUGIN_FUNC_SOCAT'     "${kn_func_socat}"
+
```

Comment:
We also need to list the ubi image. See my response here: https://github.com/openshift-knative/kn-plugin-func/pull/334/files#r989928931

Reply:
OK. I'll wait a bit on the decision for pipeline impl.

Reply:
UBI image will not be needed